src/book/03-guides/03-hardware/02-servo-calibration.mdx

```diff
@@ -0,0 +1,45 @@
+---
+section: Guides
+chapter: Hardware
+title: Servo Setup and Calibration
+description: Discover how to calibrate servos.
+slug: /guides/hardware/servo-calibration
+---
+
+## Servo Calibration
+
+Servos are electrical components that allow for the precise control and positioning of the various limbs attached to the NUgus robot. Usually, when a servo is found to not be performing correctly (i.e. seizing or a crunching sound can be heard), it is required to be removed and replaced with a new servo. 
+
+Before a servo is first used, it\'s default parameters need to be updated to ensure correct operation of the limbs. This is done using [Dynamixel Wizard](https://www.robotis.com/service/download.php?no=1670)
+
+
+To update the servo parameters, perform the following instructions:
+
+1. Connect the USB to the computer with the Dynamixel Wizard.
+![Photo of the USB attachment for the Dynamixel Wizard connected to a computer](./images/servo_calibration/usb.jpg)
+
+2. Connect the power supply and the servo to the attached component of the USB.
+![Photo of the servo cable connected to the power attachment of the USB](./images/servo_calibration/power_supply.jpg)
+
+3. Open Dynamixel Wizard
+
+4. Select Scan and wait for the attached servo to be discovered
+(OPTIONAL) Brand new servos should be preset to an ID of 1. Lower the ID range in 'Options > Scan' to speed up discovery.
+![Screenshot of Dynamixel Wizard performing a scan with no servos discovered](./images/servo_calibration/scan.png)
+![Screenshot of Dynamixel Wizard performing a scan with 3 servos discovered](./images/servo_calibration/discover.png)
+
+5. If there a multiple servos discovered, the new servo should be the option with ID = 1 and Baud Rate = 57600 bps
+
+6. Update the following parameters (don\'t forget to press save after each update):
+
+  - ID: Varies depending on where the servo will be placed. See below for guidance (TODO)
+  - Baud Rate (Bus): 1M bps = 2M/(1 + 1)
+
+  ![Screenshot of Dynamixel Wizard Baud Rate selection with 1M bps selected](./images/servo_calibration/update_parameters.png)
+
+7. Test the calibration by setting the Goal Position to 0, +90 and -90 and observing the position of the arrow on the horn.
```

Comment:
Also ensure that the servo is in joint mode by setting goal position to ~170 degrees and then to ~190 degrees and ensuring that the servo takes the long route during the position change.

Reply:
I'm assuming that you'd just need to select Joint from the dropdown menu if it's not in Joint mode?

Reply:
@Bidski do you need to manually confirm it as you say, or is the dropdown saying `Joint mode` sufficient?